apps/estuary/test/integration/application_start_test.exs

```diff
@@ -1,11 +1,68 @@
 defmodule Estuary.StartTest do
   use ExUnit.Case
+  use Placebo
   use Divo
+  import ExUnit.CaptureLog
+  require Logger
 
   @elsa_endpoint Application.get_env(:estuary, :elsa_endpoint)
   @event_stream_topic Application.get_env(:estuary, :event_stream_topic)
 
-  test "Topic is created when Estuary starts" do
-    assert Elsa.Topic.exists?(@elsa_endpoint, @event_stream_topic)
+  describe "Estuary.MessageHandler receives messages sent to the event stream" do
+    setup do
+      allow(Estuary.MessageHandler.handle_messages(any()),
```

Comment:
With this being mocked (in an integration test), it feels a lot like we're just testing the Elsa does what it says it does. Maybe we could get rid of this mocking and the test on line 25, and keep the create topic test and the debug test?

Reply:
I agree that it looks like we are testing that elsa does what it says what it does. I just don't know of a better way to test that we read the message without testing what we do with the message. I broke them up into two steps, because I think "reading the message" and "logging the message" are two different tests.

Reply:
For me the proof that you read the message (for now) is that you were able to log it.

Reply:
@jmorris-pillar what do you think?